packages/core/src/context.ts

```diff
@@ -0,0 +1,46 @@
+import { getFile } from "./utils";
+
+/**
+ * This function is responsible to add data into single block context 
+ * @param {singleBlockContext} object context of a single block
+ * @param {data} object data for the contract
+ */
+export function addDataToContext(singleBlockContext: any, data:any) {
+  if(data){
+    Object.keys(data).forEach((key: string) => {
+      singleBlockContext.data[key] = data[key];
+    });
+  }
+}
+
+/**
+ * This function is responsible to add keys into single block context from selected .keys file
+ * @param {singleBlockContext} object context of a single block
+ * @param {blockName} string block name
+ */
+export function addKeysToContext(singleBlockContext: any, ymlContent:any) {
+  const extractedKeys = getFile(ymlContent?.keysFile) || '{}';
+  const keys = JSON.parse(extractedKeys);
+  Object.keys(keys).forEach((key: string) => {
+    singleBlockContext.keys[key] = keys[key];
+  });
+}
+
+/**
+ * This function is responsible to add keys into single block context from selected .keys file
+ * @param {singleBlockContext} object context of a single block
+ * @param {blockName} string block name
+ */
+ export function addConfToContext(singleBlockContext: any, ymlContent:any) {
+  const conf = getFile(ymlContent?.confFile) || "color=0";
```

Comment:
why the default is `color=0` 🤣 it was already like that? guess correct value should be an empty string

Reply:
oki, we agreed that could be `debug=0`

Reply:
No, not debug=0 but debug=2, which is the current default

Reply:
Or better, empty string